xclim/sdba/properties.py

```diff
@@ -213,17 +214,18 @@ def spell_length_distribution(
     stat: str = "mean",
     time_res: str = "year",
 ) -> xr.DataArray:
-    r"""Spell length distribution.
+    fr"""Spell length distribution.
```

Comment:
I had no idea that this was allowed!

Reply:
It's not, it's a typo!

Reply:
Ok, phew! You blew my mind for a minute there.